### ISSUE TITLE
DoT and DoH capabilities

### ISSUE THREAD

**Issue by farzamography** [rohammosalli/smart-dns#5]
Kudos Roham! It is a great project. How can I run a DNS over TLS/HTTPS using your smart-dns?

**Comment by korvinko** [comment]
Yeah. It would be nice to make it work in android OS without additional apps only to by setting private DNS.
